shoop/core/templatetags/shoop_common.py

```diff
@@ -23,6 +24,14 @@
 from shoop.utils.serialization import ExtendedJSONEncoder
 
 
+@library.global_function
+def get_localized_language_name(code, name):
+    if code == name:
+        return get_language_info(code.lower())["name_local"]
```

Comment:
So this will always return the language's name in that language... That's one possible behavior; the other would be to return the language's name in the current locale (`get_current_babel_locale().languages.get(code.lower())`).

I'm not sure which is better here though! The current behavior lets people fluent in that language switch to that language; the other behavior would let people fluent in the current UI language to easily understand what they're going to switch to...